tests/mpi/torus.cc

```diff
@@ -76,11 +76,6 @@ main(int argc, char *argv[])
     argc, argv, testing_max_num_threads());
   MPILogInitAll log;
 
-  if (Utilities::MPI::this_mpi_process(MPI_COMM_WORLD) == 0)
-    {
-      static initlog();
```

Comment:
I don't even know what this line would have done -- isn't this a K&R-style declaration of a function?

Reply:
That's a scoped forward declaration (but can't be `static`). https://godbolt.org/z/L1Na_R

Reply:
So what does the original code do, then? I have to admit being very confused :-)

Reply:
The original code was
```
static std::ofstream logfile("output");
deallog.attach(logfile);
```
replacing everything except `static` by `initlog()` was appropriate in almost all cases. I just missed toi check if there was anything before `std::ofstream` in those lines.

Reply:
What I meant was: I don't understand what the syntax `static initlog();` does. The `static` suggests to me that it wasn't a function call to `initlog()`, but I also can't figure out what else this may have been :-) What code is actually executed here and why? :-))